Documentation/guides/building-apps/build-items.md

```diff
@@ -204,6 +204,26 @@ used to specify the ABI that the library targets. Thus, if you add
 </ItemGroup>
 ```
 
+## AndroidPackagingOptionsExclude
+
+A set if RegEx compatible items which will allow for items to be
```

Comment:
I don't think that the description matches the usage.  The example usage is `*.kotlin_*`; that is *not* a regular expression.  That's a *file glob*, though a more "consistent" description may be "search pattern", a'la [`Directory.EnumerateFiles(path, searchPattern, searchOptions)`](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.enumeratefiles?view=net-6.0#system-io-directory-enumeratefiles(system-string-system-string-system-io-searchoption)).